contracts/bonding/BondingVotes.sol

```diff
@@ -349,35 +338,41 @@ contract BondingVotes is ManagerProxyTarget, IBondingVotes {
     }
 
     /**
-     * @notice Gets the bonding state of an account at a given round.
-     * @dev In the case of delegators it is the amount they are delegating to a transcoder, while for transcoders this
-     * includes all the stake that has been delegated to them (including self-delegated).
-     * @param _account The account whose bonding state we want to get.
-     * @param _round The round for which we want to get the bonding state. Normally a proposal's vote start round.
-     * @return amount The active stake of the account at the given round including any accrued rewards. In case of
-     * transcoders this also includes all the amount delegated towards them by other delegators.
-     * @return delegateAddress The address the account delegated to. Will be equal to _account in case of transcoders.
-     */
-    function getBondingStateAt(address _account, uint256 _round)
+     * @notice Gets the voting power and delegate of an account at the start of a given round.
+     *
+     * Notice that this function is different from the {IERC5805Upgradeable} functions above that return the state at
+     * the *end* of the round. The state at the end of a round is equal to the state at the start of the next round, so
+     * to get the same result here, call this function with `round+1` instead.
+     * @dev The value returned by this can also be calculated with the following logic using BondingManager functions at
+     * the start of the corresponding round:
+     * - If `isRegisteredTranscoder(_account)`, the result is `(_account, transcoderTotalStake(_account))`
```

Comment:
The values in the return tuple here seem to be in reversed order i.e. (address, uint256) instead of (uint256, address) relative to the return tuple definition for the function.

Reply:
🤦 I remember I had the right ordering at first and went to check if it was right, but probably saw the func args instead of retval and flipped them here.